<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgMeasurePolyline</class>
 <widget class="QWidget" name="DlgMeasurePolyline">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>726</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>DetialSetup</string>
  </property>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>650</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoadDxf">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>680</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Dxf</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="gvDisplay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>811</width>
     <height>631</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtMX">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>641</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblX">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>640</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtMY">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>681</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblX_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>680</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>650</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbPLines">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>640</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDraw">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>680</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbUnit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>640</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>DlgMeasurePolyline</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>685</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
